.github/workflows/test_unit.yaml

```diff
@@ -54,6 +54,28 @@ jobs:
           export PATH=/opt/${{matrix.version}}/gcc_64/bin:$PATH
           ./scripts/tests/unit_tests.sh
 
+      - name: Install Grcov
```

Comment:
Ok.. bad news! This approach does not work because the test script "cleans up" the compilation folders. All the gdca files are gone at this point.

We need to work at a script level. I would proceed in this way:

Add a `--grcov file.lcov` param at the script doing something similar to what we did here: https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/scripts/macos/apple_compile.sh#L39-L68

if `--grcov file.lcov` is passed, runs a new function called `utest_grcov` and implement it for `linux/utils/commons.sh`, `macos/utils/commons.sh` and `windows/utils/commons.sh`. For windows, this command will be NOOP. For linux and mac, you run the grcov command to generate the file `file.lcov`.

Then, here, when the script is executed (line 55), pass: `./scripts/tests/unit_test.sh --grcov final.lcov`.

Basically, this complexity must be moved into the script.